.github/workflows/remove-label.yaml

```diff
@@ -0,0 +1,18 @@
+name: remove-label
+
+on:
+  pull_request:
+    types: [synchronize]
+
+jobs:
+  remove:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.head.repo.full_name != 'openshift-psap/special-resource-operator'
```

Comment:
@ybettan I think this should be updated.

Reply:
This is very wired, I remember I have tested in on signoz repo and it worked which make me wonder...
I will fix that one.

Filter are missing as well.

Reply:
Fixed.